dask_match/datasets.py

```diff
@@ -224,5 +262,8 @@ def timeseries(
     if seed is None:
         seed = np.random.randint(2e9)
 
-    expr = Timeseries(start, end, dtypes, freq, partition_freq, seed, kwargs)
+    projection = kwargs.pop("_projection", None)  # Not intended for public use
```

Comment:
Don't really *need* to expose this in `timeseries` at all, but it does make testing a bit simpler.